api/src/main/java/module-info.java

```diff
@@ -387,19 +389,35 @@
  *
  * <tr style="vertical-align: top; background-color:#eee"><td>{@code find}</td>
  * <td>for find operations</td>
- * <td>{@code findByHeightBetween(minHeight, maxHeight)}</td></tr>
+ * <td>{@code findByHeightBetweenOrderByAge(minHeight, maxHeight)}</td></tr>
  * </table>
  *
- * <p>The conditions are defined by the portion of the repository method name
- * (referred to as the Predicate) that follows the {@code By} keyword,
- * in the same order specified.
- * Most conditions, such as {@code Like} or {@code LessThan},
- * correspond to a single method parameter. The exceptions to this rule are
- * {@code Between}, which corresponds to two method parameters, and {@code Null},
- * {@code True}, and {@code False}, which require no method parameters.
- * Multiple conditions are delimited by the keywords {@code And}
- * and {@code Or}. The equality condition is implied when no
- * condition operator keyword is present.</p>
+ * <p>The method name must begin with an <em>action clause</em> that starts with a
+ * prefix ({@code count}, {@code delete}, {@code exists}, or {@code find}) that
+ * specifies the action of the query. The {@code find} prefix can optionally be
+ * followed by the keyword {@code First} and 0 or more digits, which limit the
+ * number of results retrieved. The remaining portion of the action clause is
+ * ignored text that must not contain reserved keywords.</p>
+ *
+ * <p>A <em>restriction clause</em> consisting of the {@code By} keyword
+ * followed by one or more conditions can optionally follow the action clause.
+ * Multiple conditions must be delimited by the {@code And} or {@code Or} keyword.
+ * Each condition consists of a case insensitive entity attribute name, optionally
+ * followed by the {@code IgnoreCase} keyword (for text properties), optionally
+ * followed by the {@code Not} keyword, optionally followed by a condition operator
+ * keyword such as {@code StartsWith}. The equality condition is implied when no
+ * condition operator keyword is present. Most of the condition operations, such as
+ * {@code Like} or {@code LessThan}, correspond to a single method parameter. The
+ * exceptions to this rule are {@code Between}, which corresponds to two method
+ * parameters, and {@code Null}, {@code True}, and {@code False}, which require
+ * no method parameters.</p>
+ *
+ * <p>A {@code find} query can optionally end with an <em>order clause</em>,
```

Comment:
```suggestion
 * <p>A {@code find} query can optionally end with an <em>order</em>,
```

We're not using the term "clause" anywhere else in the description of QbMN.